sdk/metricsadvisor/test-resources.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
FormRecognizer uses a blob storage resource id, 

https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/formrecognizer/test-resources.json#L33-L36

then get information about the blob

https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/formrecognizer/test-resources.json#L98-L105

I wonder whether we could do the similar with a Metrics Advisor resource id, then get the endpoint and subscription key.  We might not be able to get the x-api-key though...

Reply:
We are not really using the test-resources.json anymore for static resources. We will look into updating that after dynamic resources.